twingate/client_resource.go

```diff
@@ -214,7 +222,12 @@ func (client *Client) createResource(ctx context.Context, resource *Resource) (*
 		return nil, NewAPIError(ErrGraphqlResultIsEmpty, "create", resourceResourceName)
 	}
 
-	return response.ResourceCreate.Entity.convertResource(), nil
+	resource, err := client.readAllResourceGroups(ctx, response.ResourceCreate.Entity)
```

Comment:
This is not necessary on `create` - you just created it you already know all its group ids...

Reply:
I assumed that on the backend side might be some additional steps like:
- assigning some additional system groups
- filtering out/not allow to assign some groups

if you think it's not the case, then we can really simply remove this call for fetching all pages